### DISCUSSION TITLE
My Nginx Proxy Manager won't start After deleting an SSL certificate.

### DISCUSSION THREAD

**Discussion by RitZz48** [NginxProxyManager/nginx-proxy-manager#3235] [General]
So I recently changed my homelab's main domain. Every subdomain was changed from `services.example.com` to `services.example.me`

After moving the domains. I decided to delete the existing SSL cert for `*.example.com` However, unbeknown to be me. There were some dead URL that was set on `Disabled` that was still using the old domain. Now after a restart Nginx won't start and spitting this error log. 

```
❯ Starting nginx ...

nginx: [emerg] cannot load certificate "/etc/letsencrypt/live/npm-1/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/npm-1/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
```

My question is, how do I get Nginx Restarted properly without deleting the list of all my Proxy Hosts? I use a Mysql Database for NPM.

**Comment by 0088shurik8800** [comment]
Hi bro, did you solve the problem?
I meet the same((

**Comment by ghost** [comment]
Reinstall NPM (as in Nginx proxy manager) by running `sudo docker compose rm` and then running `sudo docker compose up -d`.

**Reply by 0088shurik8800** [reply]
Hi, its cant affects. The problem still exists(

**Comment by DrBorland** [comment]
I also had this issue. I fixed it by removing 'letsencrypt' volume before restarting the container.
